doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -1723,13 +1723,13 @@ This resource allows you to obtain a list of subresources published by the API:
                   "value": {
                      "type": "Polygon",
                      "coordinates": [ [
-                        [ 42.847476, -2.763969 ],
-                        [ 42.826006, -2.743151 ],
-                        [ 42.826485, -2.653740 ],
-                        [ 42.867061, -2.630934 ],
-                        [ 42.881801, -2.640617 ],
-                        [ 42.867767, -2.726723 ],
-                        [ 42.847476, -2.763969 ]
+                        [-2.763969, 42.847476],
```

Comment:
@fgalan we have updated the PR fixing fiware-ngsiv2-cookbook-rc-2016_05_31.apib as requested.